.github/workflows/conformance-clustermesh.yaml

```diff
@@ -0,0 +1,504 @@
+            ${{ github.event.issue.pull_request.url || github.event.pull_request.url }})
+          SHA=$(echo "$PR_API_JSON" | jq -r ".head.sha")
+          OWNER=$(echo "$PR_API_JSON" | jq -r ".number")
+        else
+          SHA=${{ github.sha }}
+          OWNER=${{ github.sha }}
+        fi
+
+        # bpf.masquerade is disabled due to #23283
+        CILIUM_INSTALL_DEFAULTS="--chart-directory=install/kubernetes/cilium \
+          --helm-set=image.repository=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/cilium-ci \
+          --helm-set=image.useDigest=false \
+          --helm-set=image.tag=${SHA} \
+          --helm-set=operator.image.repository=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/operator \
+          --helm-set=operator.image.suffix=-ci \
+          --helm-set=operator.image.tag=${SHA} \
+          --helm-set=operator.image.useDigest=false \
+          --helm-set=bpf.masquerade=false \
+          --config monitor-aggregation=none \
+          --rollback=false \
+          --version="
+
+        CILIUM_INSTALL_TUNNEL="--helm-set=tunnel=vxlan"
+        if [ "${{ matrix.tunnel }}" == "disabled" ]; then
+          CILIUM_INSTALL_TUNNEL="--helm-set-string=tunnel=disabled \
+            --helm-set=autoDirectNodeRoutes=true \
+            --helm-set=ipv4NativeRoutingCIDR=10.240.0.0/12 \
+            --helm-set=ipv6NativeRoutingCIDR=fd00:10:240::/44"
+        fi
+
+        case "${{ matrix.ipFamily }}" in
+          ipv4)
+            CILIUM_INSTALL_IPFAMILY="--helm-set=ipv4.enabled=true --helm-set=ipv6.enabled=false"
+            KIND_POD_CIDR_1="10.242.0.0/16"
+            KIND_SVC_CIDR_1="10.243.0.0/16"
+            KIND_POD_CIDR_2="10.244.0.0/16"
+            KIND_SVC_CIDR_2="10.245.0.0/16"
+            ;;
+          ipv6)
+            CILIUM_INSTALL_IPFAMILY="--helm-set=ipv4.enabled=false --helm-set=ipv6.enabled=true"
+            KIND_POD_CIDR_1="fd00:10:242::/48"
+            KIND_SVC_CIDR_1="fd00:10:243::/112"
+            KIND_POD_CIDR_2="fd00:10:244::/48"
+            KIND_SVC_CIDR_2="fd00:10:245::/112"
+            ;;
+          dual)
+            CILIUM_INSTALL_IPFAMILY="--helm-set=ipv4.enabled=true --helm-set=ipv6.enabled=true"
+            KIND_POD_CIDR_1="10.242.0.0/16,fd00:10:242::/48"
+            KIND_SVC_CIDR_1="10.243.0.0/16,fd00:10:243::/112"
+            KIND_POD_CIDR_2="10.244.0.0/16,fd00:10:244::/48"
+            KIND_SVC_CIDR_2="10.245.0.0/16,fd00:10:245::/112"
+            ;;
+          *)
+            echo "Unknown IP family '${{ matrix.ipFamily }}'" && false
+            ;;
+        esac
+
+        HUBBLE_ENABLE_DEFAULTS="--chart-directory=install/kubernetes/cilium \
+          --relay-image=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/hubble-relay-ci:${SHA} \
+          --relay-version=${SHA}"
+        CLUSTERMESH_ENABLE_DEFAULTS="--apiserver-image=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/clustermesh-apiserver-ci \
+          --apiserver-version=${SHA} --service-type=NodePort"
```

Comment:
`--apiserver-version` is not a parameter to specify a tag. You should do `--apiserver-image=quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/clustermesh-apiserver-ci:${SHA}`.

Reply:
For the clustermesh API server image it is actually, since it is not yet using helm under the hood: https://github.com/cilium/cilium-cli/blob/fc94d2015cf9d33f087c53760dfd8f3aefffd8de/internal/utils/utils.go#L78-L80

This is also confirmed by the output of the installation command: https://github.com/cilium/cilium/actions/runs/4235266535/jobs/7358697796#step:16:18